builder/virtualbox/iso/builder.go

```diff
@@ -89,13 +89,19 @@ type Config struct {
 	// The type of controller that the primary hard drive is attached to,
 	// defaults to ide. When set to sata, the drive is attached to an AHCI SATA
 	// controller. When set to scsi, the drive is attached to an LsiLogic SCSI
+	// controller. When set to pcie, the drive is attached to an NVMe
```

Comment:
This worked for me but I had to install an [extension pack](https://www.virtualbox.org/wiki/Downloads#VirtualBox6.0.14OracleVMVirtualBoxExtensionPack), may be we should document that too ?